### ISSUE TITLE
content-type missing in HTTP POST

### ISSUE THREAD

**Issue by aristipp** [wkhtmltopdf/wkhtmltopdf#1526]
If a try to send a post value this error occurs:

    content-type missing in HTTP POST, defaulting to application/x-www-form-urlencoded. Use QNetworkRequest::setHeader() to fix this problem. Exit with code 1 due to http error: 1203

I used wkhtmltopdf 0.12 on Linux 64bit:
`wkhtmltopdf --post 'pdf' 'true' http://...`

**Comment by BCSTECH** [comment]
hi,

Is this already solved?..what is the solution for content-type missing HTTP Post issue?..I can't any solution to solve this. Can you help me with this?..

Thanks,
Kevin

**Comment by ashkulz** [comment]
Did you try with the latest version from the downloads page?

**Comment by jboilesen** [comment]
Yes, I tried and I'm still getting this error (wkhtmltopdf current stable 0.12.2.1 for debian jessie):
Command to reproduce:
wkhtmltopdf 'http://httpbin.org/post' --post var1 value1 --post var2 value2 /home/jow/post.pdf

**Comment by jboilesen** [comment]
PS: (thanks for wkhtmltopdf)

**Comment by omkanwar** [comment]
Is this issue fixed? I still see the issue on my installation on Ubuntu 16.04 LTS. I get following message and post params are not sent:

content-type missing in HTTP POST, defaulting to application/octet-stream

I tried a sample command line like this:

wkhtmltopdf-amd64   --post g gggg --post g sds http://www.mysite.com/myurl.php  mypdf.pdf

**Comment by Rakesh-Sharma-H-M** [comment]
It doesn't work with latest version as well.

Cmd: wkhtmltopdf --cookie-jar my.jar --post uid admin --post passw admin --post btnSubmit Login http://localhost:4502/content/medtronic-com/global/index.html test123.pdf

**Comment by TheSin-** [comment]
is this still an issue? is there a work around?

mine does
```
Loading page (1/2)
content-type missing in HTTP POST, defaulting to application/x-www-form-urlencoded. Use QNetworkRequest::setHeader() to fix this problem.
[==============================>                             ] 51%
```

and just sits there, this is Debian jessie updated to today, which is using wkhtmltopdf 0.12.1
